packages/dds/tree/api-extractor/api-extractor.current.json

```diff
@@ -0,0 +1,5 @@
+{
+	"$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+	"extends": "<projectFolder>/../../../common/build/build-common/api-extractor-report.esm.current.json",
+	"mainEntryPointFilePath": "<projectFolder>/lib/public.d.ts"
+}
```

Comment:
Should be beta.d.ts - it should be the least stable export. Currently that is beta.
https://github.com/microsoft/FluidFramework/wiki/Maintaining-API-support-levels